WCSim.mac

```diff
@@ -350,6 +350,22 @@
 #/gps/direction 1 0 0
 #/gps/position 0 0 0
 
+# Or inject photon from mPMT LED
+#/mygen/generator mPMT-LED
+# Use PMTid (from 1 to totalNum_mPMTs) to specify source mPMT
+# Use LEDid (from 0 to 11) to specify LED position, theta and phi directional change from nominal direction
```

Comment:
Naive question here. I thought the LED mPMT proposal was to replace 5 (of the 19) 3" PMTs each with 8 LEDs = 40 LEDs in total. Why the number 12 here?

Reply:
This is another set of calibration LEDs which will be placed between the small PMTs

Reply:
The LEDs are the green dots in the figure.
![image](https://github.com/WCSim/WCSim/assets/19830271/edd3655c-a334-4a4c-aee1-ef66cd100ec8)

Reply:
Ahh I see, thanks. The difference between "mPMT LED" and "LED-mPMT". Not confusing at all... 
They're the names we use in the collaboration, so we have to use them. But what we can do is be explicit to avoid confusion. Something like
```
# Or inject photon from mPMT LED
# (Note that these are LEDs positioned in the standard mPMT modules.
#  They are not LEDs in the LED-mPMT modules)
```

Reply:
done